FAQ.md

```diff
@@ -5,6 +5,10 @@
 specified in the restored PVC, as we don't rely on the StorageClass for the PV placement on restore in the release v1.0.0.
 A fix to the PV placement based on the StorageClass can be expected in a future release.
 
+3. Pulling data manager image will fail if there are multiple "/" in the image name. velero-plugin-for-vsphere v1.0.1 only 
```

Comment:
If this is fixing this issue why are we adding this into the FAQ?

Reply:
Removed from FAQ.